core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceComponent.scala

```diff
@@ -598,6 +687,42 @@ trait WorkspaceComponent {
       }
     }
 
+    def findWorkspaceUsersAndCatalog(workspaceId: UUID): ReadAction[Set[(Either[RawlsUserRef, RawlsGroupRef], Boolean)]] = {
```

Comment:
Shares some common code with `findWorkspaceUsersAndAccessLevel` - can that be split out into new functions or methods?